### DISCUSSION TITLE
How to use a specified image as the home page background, rather than the primary color defined in the /src/css/custom.css?

### DISCUSSION THREAD

**Discussion by bonjour-npy** [facebook/docusaurus#9003] [Q&A]
How to use a specified image as the home page background, rather than the primary color defined in the /src/css/custom.css?
Whenever I changed the primary color, the home page background changed as well, it makes my home page so ugly

**Comment by slorber** [comment]
You can use any CSS/JSX you want in your homepage and swizzle the default layout component.

I don't understand what prevents you from achieving your goal. A repro would help
